ts/components/messages/__tests__/MessageDeadlines.test.ts

```diff
@@ -0,0 +1,281 @@
+          time_to_live: (3600 as unknown) as number &
+            WithinRangeInteger<3600, 604800>
+        }
+      }
+    },
+    {
+      meta: {
+        created_at: setDate(0, 3),
+        fiscal_code: fiscalCode,
+        id: "01DQQGBXWSCNNY44CH2QZ95J7A",
+        sender_service_id: "01DP8VSP2HYYMXSMHN7CV1GNHJ",
+        time_to_live: (3600 as unknown) as number &
+          WithinRangeInteger<3600, 604800>
+      },
+      isRead: false,
+      isArchived: false,
+      message: {
+        kind: "PotSome",
+        value: {
+          content: {
+            subject: "[pagoPaTest] payment 2" as WithinRangeString<10, 121>,
+            markdown: "demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo" as WithinRangeString<
+              80,
+              10001
+            >,
+            due_date: setDate(0, 15),
+            payment_data: {
+              amount: (1 as unknown) as number &
+                WithinRangeInteger<1, 9999999999>,
+              notice_number: "002718270840468918" as string &
+                IPatternStringTag<"^[0123][0-9]{17}$">,
+              invalid_after_due_date: true
+            }
+          },
+          created_at: setDate(0, 3),
+          fiscal_code: fiscalCode,
+          id: "01DQQGBXWSCNNY44CH2QZ95J7A",
+          sender_service_id: "01DP8VSP2HYYMXSMHN7CV1GNHJ",
+          time_to_live: (3600 as unknown) as number &
+            WithinRangeInteger<3600, 604800>
+        }
+      }
+    },
+    {
+      meta: {
+        created_at: setDate(1, 12),
+        fiscal_code: fiscalCode,
+        id: "01DQQGBXWSCNNY44CH2QZ95PIO",
+        sender_service_id: "01DP8VSP2HYYMXSMHN7CV1GNHJ",
+        time_to_live: (3600 as unknown) as number &
+          WithinRangeInteger<3600, 604800>
+      },
+      isRead: false,
+      isArchived: false,
+      message: {
+        kind: "PotSome",
+        value: {
+          content: {
+            subject: "[pagoPaTest] payment 2" as WithinRangeString<10, 121>,
+            markdown: "demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo demo" as WithinRangeString<
+              80,
+              10001
+            >,
+            due_date: setDate(1, 12),
+            payment_data: {
+              amount: (1 as unknown) as number &
+                WithinRangeInteger<1, 9999999999>,
+              notice_number: "002718270840468918" as string &
+                IPatternStringTag<"^[0123][0-9]{17}$">,
+              invalid_after_due_date: true
+            }
+          },
+          created_at: setDate(1, 12),
+          fiscal_code: fiscalCode,
+          id: "01DQQGBXWSCNNY44CH2QZ95PIO",
+          sender_service_id: "01DP8VSP2HYYMXSMHN7CV1GNHJ",
+          time_to_live: (3600 as unknown) as number &
+            WithinRangeInteger<3600, 604800>
+        }
+      }
+    }
+  ]
+};
+
+describe("last id check", () => {
+  it("should return true", async () => {
```

Comment:
```suggestion
const sections = await Promise.resolve(generateSections(messagesState));
  it("should return true", async () => {
```
sections could be loaded only a time, at the beginning